practice-one/src/index.html

```diff
@@ -44,6 +45,64 @@
+                  fill="white" />
+              </svg>
+              Download
+            </a>
+            <a href="#" class="btn btn-link btn-download">
+              <svg class="download-icon" xmlns="http://www.w3.org/2000/svg" width="41" height="41" viewBox="0 0 41 41"
+                fill="none">
+                <path
+                  d="M20.5003 37.5832C11.0652 37.5832 3.41699 29.935 3.41699 20.4998C3.41699 11.0647 11.0652 3.4165 20.5003 3.4165C29.9355 3.4165 37.5837 11.0647 37.5837 20.4998C37.5734 29.9298 29.9303 37.5729 20.5003 37.5832ZM13.2006 25.261C12.9802 25.261 12.7616 25.2883 12.5463 25.3395L12.2901 25.4096L11.2753 27.147C11.09 27.4561 11.0305 27.8247 11.1091 28.1765C11.1877 28.5283 11.3984 28.8365 11.6978 29.0373C11.9971 29.2382 12.3621 29.3164 12.7175 29.2558C13.0728 29.1953 13.3913 29.0005 13.6072 28.7118L13.7131 28.5529L15.0627 26.2108C14.844 25.9183 14.5605 25.6803 14.2345 25.5155C13.9085 25.3507 13.5488 25.2636 13.1835 25.261H13.2006ZM22.5691 14.794C22.0376 15.2933 21.6852 15.9537 21.5663 16.6732C21.3759 17.6653 21.5584 18.6927 22.0788 19.5585L27.2722 28.5546C27.4529 28.8587 27.7411 29.084 28.0799 29.1858C28.4187 29.2877 28.7833 29.2587 29.1018 29.1047C29.4193 28.9481 29.6661 28.6774 29.7928 28.3468C29.9195 28.0162 29.9167 27.6499 29.7851 27.3212L29.7014 27.1504L28.3347 24.7741H30.3677C30.7237 24.7711 31.0653 24.6332 31.3236 24.3881C31.5819 24.1431 31.7376 23.8092 31.7592 23.4538C31.7809 23.0984 31.667 22.7481 31.4404 22.4734C31.2138 22.1988 30.8915 22.0204 30.5385 21.9741L30.3677 21.9639H26.7118L22.9637 15.4722L22.5725 14.7889L22.5691 14.794ZM10.6945 21.9519C10.5109 21.9524 10.3292 21.989 10.1598 22.0598C9.99046 22.1306 9.8367 22.2341 9.70736 22.3643C9.57803 22.4946 9.47567 22.6491 9.40615 22.819C9.33663 22.9889 9.3013 23.1709 9.3022 23.3545C9.3022 24.0617 9.82837 24.6596 10.5305 24.7502L10.7013 24.7587H23.4797C23.6211 24.4747 23.6922 24.161 23.6872 23.8438C23.6821 23.5266 23.601 23.2152 23.4506 22.9359C23.3227 22.6742 23.1321 22.4483 22.8958 22.2781C22.6594 22.1079 22.3846 21.9988 22.0959 21.9605L21.8892 21.9502H17.5312L23.5582 11.514C23.7439 11.1905 23.7942 10.8067 23.6983 10.4463C23.6301 10.1933 23.4926 9.96445 23.3014 9.78538C23.1102 9.6063 22.8727 9.48415 22.6158 9.43267C22.359 9.38118 22.0928 9.40241 21.8473 9.49396C21.6019 9.58551 21.3868 9.74375 21.2264 9.95088L21.1205 10.108L20.5003 11.1911L19.887 10.108C19.7063 9.80352 19.4177 9.57798 19.0786 9.4761C18.7394 9.37423 18.3743 9.40344 18.0557 9.55796C17.7383 9.71521 17.4918 9.98621 17.3651 10.317C17.2385 10.6478 17.2411 11.0142 17.3724 11.3432L17.4561 11.514L18.874 14.0082L14.2871 21.9519H10.6945Z"
+                  fill="white" />
+              </svg>
+              Download
+            </a>
+          </div>
+
+        </section>
+
+        <div class="download-background">
```

Comment:
Naming convention is not good. Additionally, let's use HTML semantic tags instead of `div` here